kolibri/core/assets/src/views/AppError/TechnicalTextBlock.vue

```diff
@@ -20,6 +20,7 @@
       <KButton
         v-if="clipboardCapable"
         ref="copyButton"
+        :style="{ marginTop: '8px', marginBottom: '8px' }"
```

Comment:
note that since nothing is dynamic here, this could have been written in pure css:

```
style="margin-top: 8px; margin-bottom: 8px;"
```